### PR TITLE
windows-ce-migration-faq.md converted into yml

### PR DESCRIPTION
**Pull request by v-hearya**

### CODE REVIEW

**Comment by v-hearya** [comment]
@TerryWarwick - Could you please review the .yml converted page and let me know if it's good to sign-off.

Preview URL - https://review.docs.microsoft.com/en-us/windows/iot-core/windows-ce-migration-faq?branch=pr-en-us-1657

cc: @Khairunj

**Comment by Khairunj** [comment]
@TerryWarwick Can you please approve the PR so we can merge it please.  I have also sent a note to you yesterday.

**Review by TerryWarwick — CHANGES_REQUESTED** [review]
Once you address the current conflicts I will complete the review.

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -167,25 +167,25 @@
 
 {
     "source_path": "windows-iotcore/windows-iot-enterprise.md",
-    "redirect_url": "https://docs.microsoft.com/en-us/windows/iot/iot-enterprise/getting_started",
+    "redirect_url": "/en-us/windows/iot/iot-enterprise/getting_started",
```
Comment (TerryWarwick) [inline .openpublishing.redirection.json:170]: Remove "en-us" to support localized version

File: .openpublishing.redirection.json [hunk 2]
```diff
@@ -167,25 +167,25 @@
...
     "redirect_document_id": false
 },
 
 {
     "source_path": "windows-iotcore/windows-server",
-    "redirect_url": "https://docs.microsoft.com/en-us/windows/iot/server/windows-server",
+    "redirect_url": "/en-us/windows/iot/server/windows-server",
```
Comment (TerryWarwick) [inline .openpublishing.redirection.json:176]: Remove "en-us" to support localized version

File: .openpublishing.redirection.json [hunk 3]
```diff
@@ -167,25 +167,25 @@
...
     "redirect_document_id": false
 },
 
 {
     "source_path": "windows-iotcore/windows-ce-app-container",
-    "redirect_url": "https://docs.microsoft.com/en-us/windows/iot/windows_ce/windows-ce-app-container-overview",
+    "redirect_url": "/en-us/windows/iot/windows_ce/windows-ce-app-container-overview",
```
Comment (TerryWarwick) [inline .openpublishing.redirection.json:182]: Remove "en-us" to support localized version

File: .openpublishing.redirection.json [hunk 4]
```diff
@@ -167,25 +167,25 @@
...
     "redirect_document_id": false
 },
 
 {
     "source_path": "windows-iotcore/windows-ce-migration-faq",
-    "redirect_url": "https://docs.microsoft.com/en-us/windows/iot/windows_ce/windows-ce-migration-faq",
+    "redirect_url": "/en-us/windows/iot/windows_ce/windows-ce-migration-faq",
```
Comment (TerryWarwick) [inline .openpublishing.redirection.json:188]: Remove "en-us" to support localized version

File: windows-iotcore/index.yml [hunk 5]
```diff
@@ -16,20 +16,20 @@ landingContent:
...
-            url: https://docs.microsoft.com/windows-hardware/manufacture/iot/iotcoreservicesoverview
+            url: /windows-hardware/manufacture/iot/iotcoreservicesoverview
             
   - title: Get started
     linkLists: 
       - linkListType: tutorial
         links: 
           - text: Windows 10 IoT Core Manufacturing Guide
-            url: https://docs.microsoft.com/en-us/windows-hardware/manufacture/iot/iot-core-manufacturing-guide
+            url: /en-us/windows-hardware/manufacture/iot/iot-core-manufacturing-guide
```
Comment (TerryWarwick) [inline windows-iotcore/index.yml:30]: remove "en-us" to support localization

File: windows-iotcore/index.yml [hunk 6]
```diff
@@ -16,20 +16,20 @@ landingContent:
...
           - text: Windows 10 IoT Enterprise Manufacturing Guide
-            url: https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/iot-ent-overview
+            url: /en-us/windows-hardware/manufacture/desktop/iot-ent-overview
```
Comment (TerryWarwick) [inline windows-iotcore/index.yml:32]: remove "en-us" to support localization

**Review by TerryWarwick — CHANGES_REQUESTED** [review]

**Comment by TerryWarwick** [comment]
applied link updates through separate PRs then applied md to yml conversion to this branch as well as the windows-iot-pr branch where the Windows-CE content was moved.  This PR is no longer needed

**Pull request closed without merging**